packages/clarity-js/src/layout/mutation.ts

```diff
@@ -42,7 +42,7 @@ export function start(): void {
         return value;
       } catch (error) {
         log.log(Code.CssRules, error, Severity.Info);
-        return null;
+        throw error;
```

Comment:
Please add a comment explaining the motivation for this change - summary of what you had in the PR note - so future developers have the context.

Reply:
Hi @sarveshnagpal I'm happy to add the comment here. To be clear, the reason we need to throw the error is to stay in spec with what the `insertRule` specification is.

MDN: https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet/insertRule#Restrictions
https://www.w3.org/TR/DOM-Level-2-Style/css.html - Exceptions listed here.